defaults/defaults.go

```diff
@@ -76,7 +76,7 @@ const (
 	ConnectivityCheckJSONMockImage   = "quay.io/cilium/json-mock:v1.3.0@sha256:2729064827fa9dbfface8d3df424feb6c792a0ba07117b844349635c93c06d2b"
 
 	ConfigMapName = "cilium-config"
-	Version       = "v1.10.2"
+	Version       = "v1.10.3"
```

Comment:
nitpick - this change should probably be in separate commit in case we need to revert it?

Reply:
I think this has to go together, otherwise you may end up with clusters with taints that will never be removed.

Reply:
Correct